speaksfer/settings/base.py

```diff
@@ -41,14 +42,14 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    # third-party imports
-    "drf_yasg",
+    "app.user",
```

Comment:
why did you remove the comment?

Reply:
It was conflicting with the name of the app but let me fix it

Reply:
did you fix it?

Reply:
Yes I did